app/views/shared/_eligible_region_content.html.erb

```diff
@@ -17,6 +17,18 @@
 
 <% end %>
 
+<h2 class="govuk-heading-m">Application timelines</h2>
+
+<p class="govuk-body">
+  Your teaching qualifications and experience will be assessed against strict requirements. These requirements are mandatory and must be met in full. The process is designed to ensure we only award QTS to the highest quality teachers. This means it can take time.
+</p>
+
+<p class="govuk-body">We assess 90 per cent of applications within 12 months.</p>
+
+<p class="govuk-body">Information correct as of July 2024. Timelines are based on total applications completed since February 2023.</p>
+
+<p class="govuk-body">It may take longer if we need to ask for further information or if external factors, or third parties, cause delays. This includes waiting for references, or verification of qualifications or professional standing. The time it takes to assess applications is regularly reviewed and subject to change.</p>
```

Comment:
As this content is the same as the content shown on the start page, could we move it in to a shared view so if we need to change it in the future we only need to change it in one place.